docs/automated-process.md

```diff
@@ -0,0 +1,30 @@
+# Automated Ansible Release Process
+
+## Preamble
+
+This document describes the (mostly) automated ansible community package release process.
+
+!!! note
+    Throughout this page, placeholder values in code blocks are formatted as
+    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to fill in.
```

Comment:
```suggestion
    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to specify.
```

Reply:
"Fill in" might be a little idiomatic. Prepositional verbs can be harder to understand.